packages/react/src/_popover/Popover.tsx

```diff
@@ -79,19 +84,14 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 			// Adds distance between the reference and floating element
 			// https://floating-ui.com/docs/offset
 			offset(offsetOption),
-			// Changes the placement of the floating element in order to keep it in view
-			// https://floating-ui.com/docs/flip
-			flip({ padding: DEFAULT_OFFSET }),
+			// Placing shift() before flip() in the array ensures it can do its work before flip() tries to change the placement.
+			// https://floating-ui.com/docs/flip#combining-with-shift
+			shift({ padding: MIN_SIDE_GUTTER_WIDTH }),
```

Comment:
Should this be removed too? Is this what's causing the issues?

Reply:
I should probably move both of these things to the other PR but this isn't causing the issue.

I think we were getting the [autoPlacement](https://floating-ui.com/docs/autoPlacement) behaviour with the fully conditional render.

i.e. this

>Chooses the placement that has the most space available automatically.

I'm trying to see how I can get this above strategy going with the [flip](https://floating-ui.com/docs/flip#conflict-with-autoplacement) middleware.

>flip() uses a fallback “no space” strategy. Ensures the preferred placement is kept unless there is no space left.
>
>autoPlacement() uses a primary “most space” strategy. Always chooses the placement with the most space available.